web/src/util.js

```diff
@@ -1,5 +1,98 @@
-const example = x => x
+// Definiere die Konstanten für die Klassen
+const COLOR_CLS_BOOL = 'BOOL';
+const COLOR_CLS_NUM = 'NUM';
+const COLOR_CLS_NULL = 'NULL';
+const COLOR_CLS_STR = 'STR';
+const COLOR_CLS_DEFAULT = 'DEFAULT';
 
+const JSON_SYNTAX_TYPE_CLASS_MAPPING = {
+    'boolean': COLOR_CLS_BOOL,
+    'number': COLOR_CLS_NUM,
+    'undefined': COLOR_CLS_NULL,
+    'string': COLOR_CLS_STR,
+    'object': COLOR_CLS_DEFAULT,
+};
+
+/**
+ * Returns the stringified JSON with Syntax Highlighting
+ * @param {object | Array} myJson The Object to stringify.
+ * @returns {string} The stringified Object.
+ */
+const getStringifiedJSON = (myJson, indentation) => {
+    if (myJson === null) {
+        console.error('myJson is not allowed to be null')
+        return false
+    }
+    //Incase the input is not an object
+    if (typeof myJson !== 'object') {
```

Comment:
Hier gilt das gleiche wie von oben drüber, wenn wir stringify mit einem anderem Typ als object aufrufen, dann würden wir das auch als string (natürlich mit entsprechender CLS von oben drüber) zurückgeben. Ruf einfach mal `JSON.stringify('foo')`, `JSON.stringify(666)` oder` JSON.stringify(true)` auf, dann solltest du sehen, dass das alles in Strings konvertiert wird und das sollte unsere Methode auch.